Modules/CaPTkTraining/src/CaPTkTraining.cpp

```diff
@@ -145,38 +136,140 @@ captk::Training::RunThread(
 	{
 		configuration = 4;
 	}
+	else
+	{
+		runResult.ok = false;
+		runResult.errorMessage = "Unknown configuration";
+		return runResult;
+	}
 
-	int foldsOrSamples = (configuration == 1) ?
-		folds.toInt() :
-		samples.toInt();
+	/*---- Check folds and samples. 
+	       They are set by the same variable to the algorithm ----*/
 
-	if (foldsOrSamples < 2 && configuration == 1)
+	if (folds < 2 && configuration == 1)
```

Comment:
What are 2 and 1? Is it possible to use enums for these?

Reply:
I don't want to change the legacy code for compatibility reason and they don't use enums. 

That's only for configuration, folds is an integer input value that just doesn't make sense if it is below 2